drogon_ctl/templates/model_h.csp

```diff
@@ -313,6 +313,9 @@ auto cols=@@.get<std::vector<ColumnInfo>>("columns");
             $$<<"    std::shared_ptr<"<<col.colType_<<"> "<<col.colValName_<<"_;\n";
     }
     %>
+    bool dirtyFlag_[{%cols.size()%}]={ false };
+
+  public:
```

Comment:
I prefer to use `friend class` instead of moving the dirtyFlag_ to public. like:
https://github.com/drogonframework/drogon/blob/a1d2bd168b92ee6a0d38c4e7a83c383a288d4890/orm_lib/tests/mysql/Users.h#L235-L239